tests/cross/src/test/scala/tests/hover/HoverDocSuite.scala

```diff
@@ -293,6 +293,92 @@ class HoverDocSuite extends BaseHoverSuite {
        |""".stripMargin,
   )
 
+  check(
```

Comment:
I imagine the fix takes care of case classes as well and they don't need additional tests, but maybe enums would be a good candidate for improved coverage? Not really sure how much should be tested explicitly, sorry. :slightly_smiling_face: 

Also, idea taken from your other PR: would it make sense to introduce a test for argument in a non-first parameter list? Again, sorry if it's just going overboard. :slightly_smiling_face: